include/bluetooth/services/gatts_client.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef BT_GATTS_CLIENT_H_
+#define BT_GATTS_CLIENT_H_
+
+/** @file
+ *
+ * @defgroup bt_gatts_client Bluetooth LE Generic Attribute Profile Service Client API
+ * @{
+ *
+ * @brief API for the Bluetooth LE Generic Attribute Profile (GATT) Service Client.
+ */
+
+#include <bluetooth/gatt.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/gatt_dm.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Length of the Service Changed characteristic value. */
+#define BT_GATTS_SERVICE_CHANGED_VALUE_LENGTH 4
```

Comment:
This doesn't seem like it is needed in the public header, the application does not need this when it is abstracted away with this interface.

Reply:
Will do.